### DISCUSSION TITLE
Replace issue template with YAML issue forms

### DISCUSSION THREAD

**Discussion by vurvdev** [wiremod/wire#2697] [Suggestions]
Github has more powerful YAML issue templates which have more flexibility compared to just a default blob of text.

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

**Comment by Denneisk** [comment]
I don't understand why the unnecessary parts weren't commented on the existing forms.

**Reply by Denneisk** [reply]
Guess it's a decent litmus test at least.

**Comment by per1234** [comment]
The suggested migration to issue forms was done in https://github.com/wiremod/wire/pull/3417 🎉
